USAGE.md

```diff
@@ -32,19 +50,23 @@ terminal.
 - On Linux, your distribution will provide a terminal; it can generally be opened with
 Ctrl+Shift+T or by searching for "Terminal" in your application menu.
 
-### On Docker and Non-Free Operating Systems ###
+### On OED and Non-UNIX Operating Systems ###
 
-Docker is well-supported on most free/libre operating systems, including GNU/Linux,
-FreeBSD, OpenBSD, and Plan9.
+Windows is not a supported platform. You can probably run OED there, but you will
+experience problems.
 
-Docker is not supported on Windows. Windows is incapable of supporting LXE containers,
-the technology on which Docker is based, and Microsoft
-[artifically limits](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/)
-the use of Docker for Windows, a virtualized technology.
+Docker is a Linux technology. It is based on Linux kernel features including namespaces
+and cgroups. On Mac OS, you can use [Docker for Mac](https://docs.docker.com/docker-for-mac/)
+to run OED in Docker. Microsoft [artifically limits](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/)
+the use of Docker for Windows, so you may experience problems.
 
-Docker is supported on MacOS via the [Docker for Mac](https://docs.docker.com/docker-for-mac/)
-software, a virtualized technology. You may experience issues with the `devstart` script
-which uses features that have problems on Docker for Mac to watch files for edits.
+Open Energy Dashboard universally uses UNIX filenames, which are seperated with forward
+slashes. Linux uses UNIX filenames. Mac OS uses UNIX filenames. BSD uses UNIX filenames.
+Solaris uses UNIX filenames. SmartOS uses UNIX filenames.
+
+For reasons known only to Bill Gates and God, Windows uses backwards slashes instead.
```

Comment:
Again, this has a negative tone. I think we should say that the forward slash is used in our scripts and this is not compatible with Windows OS.

Reply:
Understood, I'll reword it.